src/Arcus.Messaging.Health/Tcp/TcpHealthListenerOptions.cs

```diff
@@ -68,15 +54,15 @@ private static JsonSerializerSettings CreateDefaultSerializerSettings()
         }
 
         /// <summary>
-        /// Gets or sets the function that gets the TCP health check port, using the service object with a registered collection of instances.
+        /// Gets or sets the TCP health port on which the health report is exposed.
         /// </summary>
-        public Func<IServiceProvider, int> GetTcpHealthPort
+        public string TcpHealthPort
```

Comment:
Since it's called `TcpHealthListenerOptions`, I'd simply call this `PortConfigurationKey` or so?

Reply:
Renamed to `TcpPortConfigurationKey`